client/app/utils/jsonSchema/userOperator.ts

```diff
@@ -0,0 +1,379 @@
+    "legal_name",
+    "business_structure",
+    "physical_street_address",
+    "physical_municipality",
+    "physical_province",
+    "physical_postal_code",
+    "mailing_street_address",
+    "mailing_municipality",
+    "mailing_province",
+    "mailing_postal_code",
+    "operator_has_parent_company",
+  ],
+  properties: {
+    step_1: {
+      //Not an actual field in the db - this is just to make the form look like the wireframes
+      type: "object",
+      title: "Step 1. User Information",
+      readOnly: true,
+    },
+    first_name: { type: "string", title: "First Name" },
+    last_name: { type: "string", title: "Last Name" },
+    position_title: { type: "string", title: "Position Title" },
+    street_address: { type: "string", title: "Mailing Address" },
+    municipality: { type: "string", title: "Municipality" },
+    postal_code: { type: "string", title: "Postal Code" },
+    email: { type: "string", title: "Email Address" },
+    phone_number: { type: "string", title: "Phone Number" },
+    province: { type: "string", title: "Province" },
+    role: {
+      title: "Are you a senior officer of the operator?",
+      type: "boolean",
+      default: true,
+    },
+    // temporary handling of many-to-many fields, will be addressed in #138
+    // file: {
+    //   title: "Signed Statutory Declaration",
+    //   type: "string",
+    //   format: "data-url",
+    // },
+    step_2: {
+      //Not an actual field in the db - this is just to make the form look like the wireframes
+      type: "object",
+      title: "Step 2. Operator Information",
+      readOnly: true,
+    },
+    legal_name: { type: "string", title: "Legal Name" },
+    trade_name: { type: "string", title: "Trade Name" },
+    cra_business_number: {
+      type: "number",
+      title: "CRA Business Number",
+      readOnly: true,
+    },
+    bc_corporate_registry_number: {
+      type: "number",
+      title: "BC Corporate Registry Number",
+      readOnly: true,
+    },
+    business_structure: {
+      type: "string",
+      title: "Business Structure",
+    },
+    physical_street_address: {
+      type: "string",
+      title: "Physical Address (PA)",
+    },
+    physical_municipality: {
+      type: "string",
+      title: "PA Municipality",
+    },
+    physical_province: {
+      type: "string",
+      title: "PA Province",
+    },
+    physical_postal_code: {
+      type: "string",
+      title: "PA Postal Code",
+    },
+    mailing_street_address: {
+      type: "string",
+      title: "Mailing Address (MA)",
+    },
+    mailing_municipality: {
+      type: "string",
+      title: "MA Municipality",
+    },
+    mailing_province: {
+      type: "string",
+      title: "MA Province",
+    },
+    mailing_postal_code: {
+      type: "string",
+      title: "MA Postal Code",
+    },
+    website: { type: "string", title: "Website" },
+    operator_has_parent_company: {
+      title: "Does the operator have a parent company?",
```

Comment:
I think the conditionality in the form is backwards. If I change it to no, they don't have a parent company, then I'm prompted to enter parent company info